calm/control-example/pre-prod-review-specification.json

```diff
@@ -1,5 +1,5 @@
 {
-  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$schema": "https://raw.githubusercontent.com/finos/architecture-as-code/main/calm/draft/2024-08/meta/control.json#defs/control-requirement",
   "$id": "control-specification.json",
```

Comment:
I think anywhere we're using $id we should be using an absolute URI, I think that's what we've done everywhere else we've used them. I think this use effectively changes the base URI for the retrieval URI - https://json-schema.org/understanding-json-schema/structuring#id